### PR TITLE
feat(Forms): add `bubbleValidation` to Form.Isolation and Iterate.PushContainer to prevent the form from being submitted when there are fields with errors

### PR DESCRIPTION
**Pull request by tujoworker**
Partially solves #4072

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Form/Isolation/Isolation.tsx [hunk 1]
```diff
@@ -88,12 +98,25 @@ function IsolationProvider<Data extends JsonObject>(
...
+  //     return
+  //   }
+
+  //   const path = `/${id}`
+  //   outerContext.setMountedFieldState(path, {
+  //     isMounted: true,
+  //   })
+  //   outerContext.setFieldError(path, new Error('Form.Isolation'))
+  // }, [delegateValidation, id, outerContext])
+
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Form/Isolation/Isolation.tsx:119]: Should perhaps remove the lines of code above?

File: packages/dnb-eufemia/src/extensions/forms/Form/Isolation/IsolationDocs.ts [hunk 2]
```diff
@@ -20,6 +20,11 @@ export const IsolationProperties: PropertiesTableProps = {
     type: 'React.Ref',
     status: 'optional',
   },
+  delegateValidation: {
+    doc: 'Prevent the form from being submitted when there are fields with errors inside the Form.Isolation.',
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Form/Isolation/IsolationDocs.ts:24]: Do we need to add this to `PushContainerDocs.ts` as well?

File: packages/dnb-eufemia/src/extensions/forms/Form/Isolation/Isolation.tsx [hunk 3]
```diff
@@ -43,6 +48,10 @@ export type IsolationProviderProps<Data> = {
     isolatedData: JsonObject,
     handlerData: JsonObject
   ) => unknown
+  /**
+   * Propagate errors from the fields to the parent form.
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Form/Isolation/Isolation.tsx:52]: This text is different from what's documented in the props for `delegateValidation` in `packages/dnb-eufemia/src/extensions/forms/Form/Isolation/IsolationDocs.ts`.

**Review by langz — APPROVED** [review]

**Comment by langz** [comment]
`delegateValidation`, maybe we together could come up with a better name?
When I think of delegate, I'm thinking that it delegates the validation to someone or something, but in `delegateValidation` we don't say to what/who we delegate it to?
Is delegate the correct wording, when we don't have a "recipient" 🤔 

Do you have any thoughts @andlbrei?

**Comment by andlbrei** [comment]
`elevateValidation` or something similar?
Or something like `validateGlobally`, but with a different term than "global" perhaps...

**Review by langz — APPROVED** [review]
🫧

**Pull request merged**